### PR TITLE
Feature/my pools

### PR DESCRIPTION
**Pull request by tamirse**

### CODE REVIEW

File: src/components/MyPools/MyPools.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  const classes = useStyles();
+  const joinedPoolsAddresses = [];
+  let joinedPools = [];
+  let createdPools = [];
+
+  const poolsMetadata = useAsync(async () => {
+    return fetchAllPools();
+  }, []);
+
+  const accountPools = useAccountPoolsState(props.address);
```
Comment (leonprou) [inline src/components/MyPools/MyPools.js:46]: this should give all (and only) the joined pools, no?

Comment (tamirse) [reply src/components/MyPools/MyPools.js:46]: yes

Comment (leonprou) [reply src/components/MyPools/MyPools.js:46]: Ok I misunderstand it before..
but It's kinda strange you fetch all the pools here, but that looks like the easiest now 👍

Comment (tamirse) [reply src/components/MyPools/MyPools.js:46]: if you mean by the poolsMetadata, then yes. 
Initially I thought of creating a backend endpoint to fetch specific pools, but It's indeed easier and faster like this.
We would actually need to create 2 endpoints - one for fetching the specific pools (by contract address), and the other for fetching the pools created by user.
I preferred to go with the short solution for now.

Comment (leonprou) [reply src/components/MyPools/MyPools.js:46]: Agree, I though about this for some time and got to the same conclusions.

Comment (leonprou) [reply src/components/MyPools/MyPools.js:46]: Anyway you can merge :)

**Pull request merged**